.gitignore

```diff
@@ -0,0 +1,130 @@
+# Logs
```

Comment:
Why is there so much in here? Was this copied from somewhere else?

Reply:
Yes, this was on track "Setting up the Repo" section:  npx gitignore node command which generated the .gitigore file. I did same on Book-library so I will amend it too.